src/seedu/addressbook/data/person/Person.java

```diff
@@ -3,6 +3,7 @@
 import seedu.addressbook.data.tag.UniqueTagList;
 
 import java.util.Objects;
+import java.util.Comparator;
```

Comment:
It is good practice to have `import` alphabetically order. This is included in Travis check which you need to pass in the later stage of the module. Intellij auto-import features will make be in alphabetical order.